packages/edit-site/src/components/editor/index.js

```diff
@@ -96,11 +97,23 @@ function Editor() {
 				: null,
 			select: _select,
 			entityId: _entityId,
+			hasSelectedBlock: !! _select(
+				'core/block-editor'
+			).getSelectedBlock(),
```

Comment:
Any reason why we're not using the `hasSelectedBlock` selector?

https://github.com/WordPress/gutenberg/blob/328991ac901b0d0927b1a6e2853a28c03922a2ea/packages/block-editor/src/store/selectors.js#L378-L391

Reply:
nice! I wonder what will happen if multiple blocks are selected.

Reply:
but that seems like a better selector.

Reply:
So this doesn't work if multiple blocks are selected, but there is also a count selected blocks selector I tried out. but it also isn't quite working right 🤔

Reply:
maybe `getBlockSelectionStart` would work?  It seems the `getSelectedBlock` and `hasSelectedBlock` don't return when groups are selected, but `getBlockSelectionStart` should be truthy whether it is single or a group?

Reply:
That is cool idea! I think it would work. I ended up going with `getSelectedBlockCount > 0`, because I think it's a bit easier to understand. (And it relies on what you mentioned under the hood too)